### DISCUSSION TITLE
[Script request] Venus OS - Victron Energy Unix/Linux OS

### DISCUSSION THREAD

**Discussion by ps915** [community-scripts/ProxmoxVE#42] [Request script]
### Application Name

Venus OS

### Website

https://github.com/victronenergy/venus/wiki

### Description

Two routes to go about this it seems:

VenusOS packages on top of a Debian VM.
https://github.com/victronenergy/venus/wiki/install-venus-packages-on-Debian

Running an ARM64 VM and then getting the VenusOS raspberrypi distro installed on it.
https://dev.to/duelnm/setting-up-arm-vm-on-proxmox-ve-47a0

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
What exactly is the advantage of the OS? How up-to-date is it? Is it properly maintained? I see relatively few commits. 

The question I have is, isn't a Debian LXC with the part on it enough? Or do you really need a VM?

**Comment by ps915** [comment]
Hi Mick,

Using Venus OS on a Raspberry Pi adds versatility for DIY energy storage systems (ESS). It allows for more control over features, which would otherwise require purchasing Victron’s more expensive[ Cerbo GX](https://greenakku.de/mobil-pv/victronenergy-cerbo-gx?utm_source=google&utm_medium=pmax&utm_campaign=produkte&utm_content=catchall&gad_source=1&gclid=Cj0KCQiA_qG5BhDTARIsAA0UHSJjLBV_GALbo17R8qSQy-NTx0FVjQM3JkTibrUuZeWNEO_5mx4A8KYaAt6CEALw_wcB) device, priced at around 200 euros here in germany.

The Venus OS includes additional tools like Node-RED and Signal K Server for easier automation and integrations, which enhances flexibility and makes it highly suitable for DIY ESS projects. 
More Info here: https://www.victronenergy.com/live/venus-os:large

Running Venus OS on a virtual machine (VM) in Proxmox offers benefits, primarily in USB hardware compatibility, as VMs generally handle USB pass-through better than LXC containers.

**Comment by ps915** [comment]
There is also a new Version 3.5 incoming i just learned ^^

https://www.youtube.com/watch?v=9h4txxphYyU&ab_channel=VictronEnergy

**Comment by martincaddick** [comment]
Hi @ps915 ,
I came here for something else and then noticed your request. Initially I thought that would be an awesome idea but then thought a bit more about it. My pi that is running venus OS has 4 x USB ports that are in use. 1 x solar, 1 x BMV, 1 x Multiplus and the other is a GPS. 
Putting all of that into a mini-PC that is running proxmox is probably not going to work for me.
Cheers

**Comment by marc115uk** [comment]
Second this request

**Comment by brettwellmanmbo** [comment]
Would also appreciate this

**Comment by IvarTammelaKool** [comment]
Hello!
Is it posiible to make educational version, so i can add virtual controllers and so on?
